NOTES.md

```diff
@@ -6,13 +6,13 @@ docker buildx create --use
 ```
 
 ```sh
-docker buildx build --platform linux/arm64,linux/amd64 --tag horologger/nostr-wallet-connect:v0.4.2 --output "type=registry" .
+docker buildx build --platform linux/arm64,linux/amd64 --tag horologger/albyhub:v0.4.2 --output "type=registry" .
```

Comment:
do versions need to be updated here too? should they be consistent with the Alby Hub latest version? 

In that case should we fix the version in the dockerfile too?
`FROM ghcr.io/getalby/hub:v1.0.2`

Reply:
CC @horologger - I think we probably need to fix these versions too